src/routes/FilterInterface.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+
+	import { RecursiveTreeView, type TreeViewNode } from '@skeletonlabs/skeleton';
+  import { nodes } from '../treeContent';
+    
+let myTreeViewNodes: TreeViewNode[] = nodes;
+let checkedNodes : string[] = ['javascript'];
+let indeterminateNodes : string[] = ['programming', 'language'];
+											
+</script>
+
+<div class="dark-container w-screen flex pt-10 flex-col">
+  <h2 class ="h2 pl-16" style="text-align: left;" >Virtueller Katalog</h2>
+  <div class="side_by_side w-screen pl-2 pt-5 flex">
```

Comment:
```suggestion
  <div class="pl-2 pt-5 flex">
```

you shouldn't use _w-screen_ because we have paddings in parent containers.
_side\_by\_side_ is a zombie-class I think? doesn't do anything. also: we normally use normal dashes, not single underscores, when naming css classes.